.github/workflows/ci-build.yml

```
@@ -0,0 +1,187 @@
+name: CI
+
+on:
+  pull_request:
```

Perhaps this might fix it for PRs?
```suggestion
  pull_request:
    types: [ opened, reopened, synchronize ]
```

---

No, it'll work once it's merged. It worked for testing in my fork, it's just that it needs something in master first for the "base" repo before it'll do anything (at least, that's what seems to be the case)